staff/gustavo-rojas/course-2/task-1/learn-js/solutions.js

```diff
@@ -0,0 +1,65 @@
+/*Expresions*/
+export const rectPerimeter = (x, y) => 2 * (x + y);
+export const rectArea = (x, y) => (x * y);
+export const triArea = (x,y) => 1/2*(x*y);
+/* the value in exercises.json didnt match with the real result I fixed in exercises.json , in the last decimals of the last result*/
+/* {
+            "inputs": [ 5, 7 ],
+            "output": 75.39822368615503 (the original value was 75.39822368615502 )

```

Comment:
As we commented this is not an issue, is just that you are using an associative property to solve equation and it changes the response in the 14th decimal.